app/src/main/java/com/globant/weatherapp/mvp/model/WeatherModel.kt

```diff
@@ -1,6 +1,19 @@
 package com.globant.weatherapp.mvp.model
 
 import com.globant.weatherapp.mvp.contracts.WeatherContracts
+import com.globant.weatherapp.services.WeatherService
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+class WeatherModel(private val service: WeatherService): WeatherContracts.Model {
+
+    override fun getFiveDaysWeather(cityId: Int) {
+        service.getFiveDaysWeatherByCityId(cityId)
+            .subscribeOn(Schedulers.io())
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe({ forecast ->
+                val request = forecast
```

Comment:
what do you do with the service response?

Reply:
In a future PR I will map the response and use the data to be shown in cardsView